tor_core/config.py

```diff
@@ -136,6 +183,57 @@ class Config:
     # for the OCR bot
     OCR = True
 
+    # Name of the bot
+    name = None
+    bot_version = '0.0.0'  # this should get overwritten by the bot process
+    heartbeat_logging = False
+
+    @cached_property
+    def redis(self):
+        """
+        Lazy-loaded redis connection
+        """
+        from redis import StrictRedis
+        import redis.exceptions
+
+        try:
+            url = os.environ.get('REDIS_CONNECTION_URL',
+                                 'redis://localhost:6379')
+            conn = StrictRedis.from_url(url, db=0)
```

Comment:
If we're going to pull the URL from an environment variable, let's also pull the DB number too. In redis, each of the 16 DBs are completely and totally separate, so if we needed to run, say, duplicate keys for any insane reason we could.

Reply:
Sure! I suppose when the connection was entirely hardcoded it made sense to have the `db=0` as a kwarg, but I can shift it to the url default in the environment variable. 👍